docs/development/minimal-reproductions.md

```diff
@@ -1,49 +1,87 @@
+
+## Where to host the minimal reproduction
+
+If you can, use GitHub to host your reproduction.
+If the reproduction needs to be on GitLab or Bitbucket, that's also okay.
+
+## Creating a minimal reproduction
+
+There are two ways to create a minimal reproduction:
+
+- Start with an empty repository and copy files from your production repository
+- Start with a fork of your production repository and remove files and config
 
-### Why not use the production repository to debug?
+General steps:
 
-You might think, why is a minimal reproduction even needed?
-I already have the reproduction on my production repository and it's public.
-Why not use that to debug?
+1. Create your minimal reproduction repository on GitHub, only use GitLab or Bitbucket if really needed
+1. Use the fewest number of repository files and dependencies
+1. Reduce the Renovate config to a minimum
+1. Remove private or secret information
+1. Create a `readme.md` file that explains the current behavior and expected behavior
+1. Set the repository visibility to `public`
+1. Give us the link to the repository
 
-A production repository uses many dependencies, and can have a large list of custom rules in the Renovate configuration file.
-This makes it very hard to debug the repository, because there are many moving parts and debug statements could be triggered potentially hundreds of times.
+### Why we won't use your production repository to debug
 
-When people have a large Renovate configuration, they tend to think the root cause is simple.
-But often the bug is caused by two or more features interacting.
-Reducing the config to a minimum to reproduce the bug helps discover exactly which config elements are required to interact to trigger the bug.
+You may think:
 
-## Why we require a minimal reproduction
+> Why do you even need a minimal reproduction?
+> I already have the reproduction on my production repository and it's public.
+> Why not use that to debug?
 
-A debuggable reproduction repository is usually needed, and it makes the most sense for the issue reporter to create it.
-We prefer that you use GitHub to host your reproduction, but if it requires GitLab or Bitbucket to be reproduced then creating a public repository on those instead is fine.
+A production repository usually has:
 
-### "Making a reproduction repository is too much work for me"
+- many dependencies
+- many custom rules in the Renovate configuration file
+- many files that are not relevant
 
-Although we'd love to get down to zero reported bugs or feature requests remaining, there's a queue and we need to prioritize.
-If a bug or feature is not important enough for you to spend your time creating a simple repository for, that's a strong indicator to the Renovate team that it should be treated as a lower priority than those issues which do have a reproduction repository.
+Having lots of "moving parts" makes debugging tricky, because debug statements can be triggered hundreds of times.
```

Comment:
```suggestion
Having lots of "moving parts" makes debugging tricky, because debug break points can be triggered hundreds of times.
```

Should it be `break points` or `debug statements`? You used `break points` in the suggested text.